doc/data/messages/t/too-many-positional-arguments/details.rst

```diff
@@ -1,10 +1,11 @@
-Positional arguments work well for cases where the the use cases are
-self-evident, such as unittest's ``assertEqual(first, second, msg=None)``.
-Comprehensibility suffers beyond a handful of arguments, though, so for
-functions that take more inputs, require that additional arguments be
-passed by *keyword only* by preceding them with ``*``:
-
-.. code-block:: python
+Good APIs don’t have many positional parameters. For almost all API,
+comprehensibility suffers beyond a handful of arguments.
 
-    def make_noise(self, volume, *, color=noise.PINK, debug=True):
-        ...
+Positional arguments work well for cases where the the use cases are
+self-evident, such as unittest's ``assertEqual(first, second, "assert msg")``
+or ``enumerate(fruits, 4)`` (The second arg of ``enumerate`` is seldom used,
+so maybe you'd already prefer to read ``enumerate(fruits, start=4)``).
```

Comment:
Nice, but let's use an example that we don't have to immediately back off from. How about:
```suggestion
or ``zip(fruits, vegetables)``.